msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -373,7 +362,6 @@ HTTP/1.1 204 No Content
 |Messages with rich text|Videos|
 |Message reply chain|Announcements|
 |High throughput processing|Code snippets|
-||Adaptive cards|
```

Comment:
Was this intentional? We are supporting adaptive cards?

Reply:
Yes